lib/mayaUsd/resources/scripts/mayaUsdAddMayaReference.py

```diff
@@ -178,9 +178,16 @@ def createMayaReferencePrim(ufePathStr, mayaReferencePath, mayaNamespace,
         # If we created a group prim add the variant set there, otherwise add it
         # to the prim that corresponds to the input ufe path.
         variantPrim = groupPrim if groupPrim else mayaUsd.ufe.ufePathToPrim(ufePathStr)
-        vset = variantPrim.GetVariantSet(validatedVariantSetName)
-        vset.AddVariant(validatedVariantName)
-        vset.SetVariantSelection(validatedVariantName)
+        try:
+            vset = variantPrim.GetVariantSet(validatedVariantSetName)
+            vset.AddVariant(validatedVariantName)
+            vset.SetVariantSelection(validatedVariantName)
+        except (Tf.ErrorException):
+            errorMsgFormat = getMayaUsdLibString('kErrorCreateVariantSet')
+            errorMsg = cmds.format(errorMsgFormat,
+                                   stringArg=(str(variantPrim.GetPrimPath()), validatedVariantName, str(variantPrim.GetName())))
+            om.MGlobal.displayError(errorMsg)
+            return Usd.Prim()
```

Comment:
Trap Usd exceptions while trying to create Variant set and Variant. Issue our own error message.